### ISSUE TITLE
Music program is failing to mint ARKs and thus can't be completed

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/figgy#5860]
The params say post but the url says get? from params it looks like an issue with changing the resource state.

## Backtrace

    line 41 of [PROJECT_ROOT]/app/services/identifier_service.rb: update_metadata
    line 7 of [PROJECT_ROOT]/app/services/identifier_service.rb: mint_or_update
    line 22 of [PROJECT_ROOT]/app/change_set_persisters/change_set_persister/mint_identifier.rb: mint_identifier

[View full backtrace and more info at honeybadger.io](https://app.honeybadger.io/projects/53391/faults/95893122)

## Sudden Priority Justification

Dan's unable to get this program up while this error persists, and we suspect it'll cause further problems if they put the ARK in first somehow.

**Comment by tpendragon** [comment]
I think the problem here is that the imported ARK is the PDF ARK, so it's trying to update an ARK with `/pdf` at the end. We should strip that on import.

**Comment by tpendragon** [comment]
Fixed.

**Comment by hackartisan** [comment]
This failed again on https://figgy.princeton.edu/catalog/dec36166-c167-43a9-a858-a6ec824406d6

**Comment by tpendragon** [comment]
I suspect this error's a little different - someone put the wrong ark in the record as a placeholder prior to completing the record.

**Comment by hackartisan** [comment]
Yes, I should have mentioned that -- I reopened because I wasn't sure what to do about it! Does it just not require action?

**Comment by hackartisan** [comment]
Figgy is trying to update the ark in ezid but the ark isn't valid, so it's failing. Things we could do:

- not propagate arks that don't have `ark://` in them (don't pull them in from alma record) 
- ask users not to do it this way?
- allow the record to complete if the value that was in there has a failure, minting a new ark
  - there would be 2 arks, 1 would be bad forever.

How can we tell someone to fix this?
- Change the record in alma, pull in the metadata again, and then you'd be able to complete in figgy?
  - but we think maybe it won't override a local ark value once it's there.
- Change it in alma, delete the resource in figgy, then reingest it.
- Dev fix it.

**Comment by tpendragon** [comment]
Fix it with this: don't propagate arks from Orangelight that don't have ark:// in them (don't pull them in from alma record)
